src/main/java/uk/gov/companieshouse/presenter/account/model/PresenterAccountName.java

```diff
@@ -18,11 +19,17 @@ public PresenterAccountName(final String forename, final String surname) {
     }
 
     private String validateName(final String name, final int maxLength) {
+        // CHS user profiles often don't have associated forename and surnames
+        // Therefore null is a valid value for the name field.
+        if (name == null) {
+            return null;
+        }
+
         if (StringValidator.validateString(name, maxLength)) {
```

Comment:
Would this be cleaner as one line like this: 
```suggestion
        if (name == null || StringValidator.validateString(name, maxLength)) {
```